Week2/answers_queries.js

```diff
@@ -0,0 +1,106 @@
+/* -- mysql> source C:/Administrator/......../databases/week2/world.sql;
+ */
+
+const mysql = require('mysql');
+const util = require('util');
+const prompt = require('prompt');
+
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'new_world',
+});
+
+const execQuery = util.promisify(connection.query.bind(connection));
+const input = util.promisify(prompt.get.bind(this));
+
+function questions() {
+  console.log(`
+  select one the following questions with the question number:
+  
+  1. What is the capital of country X ? (Accept X from user)
+2. List all the languages spoken in the region Y (Accept Y from user)
+3. Find the number of cities in which language Z is spoken (Accept Z from user)
+4. Accept the region and language from the user.
+   Are there any countries in this region with the given language
+   as the official language ?
+   If yes, display those countries.
+   If no, display FALSE.
+5. List all the continents with the number of languages spoken in each continent
+  `);
+}
+
+const query1 =
+  'SELECT city.Name FROM city JOIN country ON city.ID = country.Capital WHERE country.Name = ?';
+const query2 =
+  'SELECT country.Name, Language FROM  countrylanguage JOIN country ON CountryCode = Code WHERE Region = ?';
```

Comment:
this would print for example:
```
Netherlands, Dutch
Belgium, Dutch
Germany, German
Austria, German
```
..
What I would like to see is simply:
```
Dutch,
German
```

Reply:
It will be better if i use here DISTINCT. 
'SELECT DISTNCT Language FROM  countrylanguage JOIN country ON CountryCode = Code WHERE Region = ?'

Reply:
Exactly